config/secrets.yml.example

```diff
@@ -15,7 +15,6 @@ development:
     delivery_threshold: 5
     delivery_interval: 5
     schema_url: 'http://localhost:8081/'
-    schemas_path: 'schemas'
```

Comment:
@jpslav by removing this, it causes the app to crash (avro_turf expects the schemas_path to be set). 

```
class AvroTurf::SchemaStore

  def initialize(path: nil)
    @path = path or raise "Please specify a schema path"
    @schemas = Hash.new
    @mutex = Mutex.new
  end
```

```
/Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/bundler/gems/avro_turf-6ef7a6b5458a/lib/avro_turf/schema_store.rb:4:in `initialize': Please specify a schema path (RuntimeError)
	29: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	28: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	27: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/railties-5.2.4.3/lib/rails/commands.rb:18:in `<main>'
	26: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/railties-5.2.4.3/lib/rails/command.rb:46:in `invoke'
	25: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/railties-5.2.4.3/lib/rails/command/base.rb:69:in `perform'
	24: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/thor-1.0.1/lib/thor.rb:392:in `dispatch'
	23: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
	22: from /Users/steveburkett/.rvm/gems/ruby-2.6.6@event-capture-api/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
```